### ISSUE TITLE
Decay Adam learning rate

### ISSUE THREAD

**Issue by hvy** [hvy/chainer-began#2]
Decay the optimizer learning rate during training. In the original paper, they seem to divide it by two when the M_global convergence stalls.

**Comment by hvy** [comment]
Fixed.
